<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="businessPage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>811</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="widget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>811</width>
       <height>707</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Business Page</string>
     </attribute>
     <widget class="QGroupBox" name="locationBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>341</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Select Location</string>
      </property>
      <widget class="QComboBox" name="stateList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;State&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="currentText">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="stateLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QLabel" name="cityLabel">
       <property name="geometry">
        <rect>
         <x>125</x>
         <y>20</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
      <widget class="QLabel" name="zipLabel">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>20</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Zipcode</string>
       </property>
      </widget>
      <widget class="QTableView" name="cityList">
       <property name="geometry">
        <rect>
         <x>115</x>
         <y>50</y>
         <width>101</width>
         <height>140</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="zipList">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>50</y>
         <width>101</width>
         <height>140</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="categoryBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>781</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Filter On Categories</string>
      </property>
      <widget class="QTableView" name="categoryList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>101</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="categoryLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Category</string>
       </property>
      </widget>
      <widget class="QPushButton" name="searchButton">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clearButton">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QTableView" name="businessGrid">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>561</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>431</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Zipcode Statistics</string>
      </property>
      <widget class="QTableView" name="topCategoryList">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>50</y>
         <width>191</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="averageIncomeLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Average Income</string>
       </property>
      </widget>
      <widget class="QLabel" name="numBusinessLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string># of Businesses:</string>
       </property>
      </widget>
      <widget class="QLabel" name="populationLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Total Population: </string>
       </property>
      </widget>
      <widget class="QLabel" name="numBusinessLabel_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Top Categories</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="refreshButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="popularZipcodeBox">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>440</y>
        <width>351</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Popular Businesses (in zipcode)</string>
      </property>
      <widget class="QTableView" name="popularBusinesses">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>331</width>
         <height>171</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="successfulZipcodeBox">
      <property name="geometry">
       <rect>
        <x>435</x>
        <y>440</y>
        <width>361</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Successful Businesses (in zipcode)</string>
      </property>
      <widget class="QTableView" name="successfulBusinesses">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>331</width>
         <height>171</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBusiness_Analytics">
    <property name="title">
     <string>Business Analytics</string>
    </property>
   </widget>
   <addaction name="menuBusiness_Analytics"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
